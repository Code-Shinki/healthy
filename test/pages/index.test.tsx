import { cleanup, render } from '@testing-library/react'
import React from 'react'
import Home from '../../src/pages/index'

describe(`TOP PAGE`, () => {
  afterEach(() => {
    cleanup
  })

  test('snapshot', async () => {
    const { asFragment } = render(<Home />)
    const tree = asFragment()
    expect(tree).toMatchSnapshot()
  })

  test('integration', async () => {
    render(<Home />)
  })
})
