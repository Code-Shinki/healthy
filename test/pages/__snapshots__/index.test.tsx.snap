// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TOP PAGE snapshot 1`] = `
<DocumentFragment>
  <ul>
    <li>
      <a
        href="/login"
      >
        /login
      </a>
    </li>
    <li>
      <a
        href="/signup"
      >
        /signup
      </a>
    </li>
    <li>
      <a
        href="/dashboard"
      >
        /dashboard
      </a>
    </li>
    <li>
      <a
        href="/checkup"
      >
        /checkup
      </a>
    </li>
  </ul>
</DocumentFragment>
`;
